_docs/style/text-conventions/voice-and-tone.md

```diff
@@ -0,0 +1,91 @@
+
+The primary job of the words in a UI is to help users complete tasks with no
+confusion and minimal interruption. However, the voice and tone that we use
+with words in a UI also influence how people think and feel about Rackspace.
+
+Voice is our style, our point of view, and our personality. Tone is our
+attitude and how we are perceived by those who read (and hear) our words. Voice
+determines how we write. Tone determines our attitude to the subject and
+audience, and how we're perceived by our audience.
+
+The words we choose — and the ways we use them — should reflect our goals: building trust, inspiring confidence, making things easier, and developing a relationship with Rackspace users.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+## Best practices
+
+Consider the following voice and tone best practices when you write in-product text:
+
+{% figure [caption:"Voice and tone attributes"] [class:"image bg-light border"] %}
+| Description | Use | Don't use |
+| --- | --- | --- |
+| Write in a way that the user wants to be spoken to. Use helpful words and phrases that are informative, simple, clear and easy to understand. | The IP address of the server still needs to be entered. Click the My Server button to complete the configuration. | The monitoring engine couldn't connect to the server. Make sure the information is correctly entered. |
+| Temper the enthusiasm conveyed in confirmation messages. | You have successfully configured databases for monitoring. | Congratulations!! ALL of your databases are now being monitored!! |
+| Be careful about laying blame. Don't take the blame for a negative situation. Don't lay the blame of the negative situation on the user. | The system could not validate your account. Please enter your credentials again. | Either you didn't enter your credentials correctly, or the system timed out. Please enter your credentials again. |
+| In positive situations, be encouraging and offer next steps. Don't take credit for the user's success. | Congrats! You can now commission and decommission servers on-demand. | Congrats! Our system is so easy to use that you are now able to commission and decommission servers on-demand. |
+| In negative situations, be clear about the problem and how the user can fix it. Don't ask the user to trust us without providing more information. | Your account details and database IP address range don't match. Ensure that the IP addresses on the Account Details page are correct. | The database configuration failed. |
+{: class="hxTable"}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+## Voice and tone attributes
```

Comment:
this seems like just a list of descriptors of writing. What's the rationale for this particular break down into voice and tone? 
how do i meaningfully translate this into better copy?
I feel like we can give more useful direct instruction.

Reply:
I agree, we can talk about this table. I don't feel like it adds much value, too abstract- maybe we can think of something...